packages/ckeditor5-package-generator/lib/index.js

```diff
@@ -66,6 +68,18 @@ module.exports = async function init( packageName, options ) {
 		'Example: ' + chalk.gray( packageManager + ' run start' ),
 		''
 	].join( '\n' ), { startWithNewLine: true } );
+
```

Comment:
In order to avoid changing the box manually in the future when we want to change displayed text, please use the [table](https://www.npmjs.com/package/table) package in order to generate the box dynamically.

Reply:
```
const tableData = [
	[ '' ],
	[ 'Supporting a wider range of CKEditor 5 versions requires using' ],
	[ 'a more complex method of importing modules from CKEditor 5.' ],
	[ '' ],
	[ 'Read more here: <LINK TO THE GUIDE EXPLAINING THIS></LINK>' ],
	[ '' ]
];

const tableConfig = {
	header: { content: 'DEVELOPMENT SERVER' },
	columns: [ {
		paddingLeft: 2,
		paddingRight: 2
	} ],
	singleLine: true,
	border: {
		topLeft: '╭',
		topRight: '╮',
		bottomLeft: '╰',
		bottomRight: '╯',
		topBody: '─',
		bottomBody: '─',
		bodyLeft: '│',
		bodyRight: '│'
	}
};

logger.info( table( tableData, tableConfig ), { startWithNewLine: true } );
```

Reply:
IMO the cost of maintaining this is low, and the `table` plugin comes with 17 other dependencies. That's quite a lot of dependencies for drawing a table.

https://npmgraph.js.org/?q=table